network/omaha/protocol.go

```diff
@@ -48,6 +48,8 @@ type Request struct {
 
 func NewRequest() *Request {
 	return &Request{
+		// silly hack, xml's Encoder doesn't want to set this.
```

Comment:
Ah, ok. Well this has nothing to do with the encoder which obviously has no business modifying the structure given to it. So either update the comment to clarify that it is required to ensure consistency with the output of the decoder or add it to the test itself since a full struct comparison is the only place this would ever matter.